Lib/ufo2ft/filters/propagateAnchors.py

```diff
@@ -62,13 +62,19 @@ def _propagate_glyph_anchors(glyphSet, composite, processed):
     anchor_names = set()
     to_add = {}
     for component in composite.components:
-        glyph = glyphSet[component.baseGlyph]
-        _propagate_glyph_anchors(glyphSet, glyph, processed)
-        if any(a.name.startswith('_') for a in glyph.anchors):
-            mark_components.append(component)
+        try:
+            glyph = glyphSet[component.baseGlyph]
+        except KeyError:
+            logger.warn(
```

Comment:
we should use `logger.warning`, not `logger.warn`
https://docs.python.org/3/library/logging.html#logging.warning